app/controllers/item.js

```diff
@@ -81,6 +83,11 @@ export default Ember.Controller.extend({
       this.set('previewUrl', image.get("previewImageUrl"));
     },
 
+    goToStockItem(inventoryNumber) {
+      let finalUrl = config.APP.BASE_HOST_URL + "/items/" + inventoryNumber;
```

Comment:
Do we really need this variable here??

Reply:
Passing long string as a parameter looks ugly, so assigned it to a variable.